recipes/r-seasonal/meta.yaml

```
@@ -0,0 +1,86 @@
+{% set version = '1.7.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-seasonal
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/seasonal_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/seasonal/seasonal_{{ version }}.tar.gz
+  sha256: 29e8d7c7a01981f52cce5478a03b5ee5074add8af35452446dbd7e1221a0cd27
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: True  # [osx]
```

Does not contain compiled code, so the skip can be removed (noarch packages are build on linux after feedstock conversion):
```suggestion
  noarch: generic
```